pkg/ingestor/parser/open_vex/parser_open_vex.go

```diff
@@ -0,0 +1,152 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package open_vex
+
+import (
+	"context"
+	"fmt"
+
+	json "github.com/json-iterator/go"
+	"github.com/openvex/go-vex/pkg/vex"
+
+	"github.com/guacsec/guac/pkg/assembler"
+	"github.com/guacsec/guac/pkg/assembler/clients/generated"
+	"github.com/guacsec/guac/pkg/assembler/helpers"
+	"github.com/guacsec/guac/pkg/handler/processor"
+	"github.com/guacsec/guac/pkg/ingestor/parser/common"
+)
+
+var (
+	justificationsMap = map[string]generated.VexJustification{
+		"component_not_present":                             generated.VexJustificationComponentNotPresent,
```

Comment:
I think you can use these enums here? https://github.com/openvex/go-vex/blob/main/pkg/vex/justification.go#L12-L49

Reply:
Done. Is this what you were talking about?

``` go
for _, justification := range vex.Justifications() {
	if justification == string(vexStatement.Justification) {
		vd.VexJustification = generated.VexJustification(justification)
	}
}
```

Reply:
No, I think @lumjjb was saying to replace the strings in the map with the enums found in go-vex.

For example:
```
justificationsMap = map[vex.Justification]generated.VexJustification{
		vex.ComponentNotPresent:  generated.VexJustificationComponentNotPresent,
}
```

Reply:
this would be a better and faster approach instead of looping through the strings you have currently. Can do the same thing for status: https://github.com/openvex/go-vex/blob/main/pkg/vex/status.go#L12-L24

Reply:
> No, I think @lumjjb was saying to replace the strings in the map with the enums found in go-vex.
> 
> For example:
> 
> ```
> justificationsMap = map[vex.Justification]generated.VexJustification{
> 		vex.ComponentNotPresent:  generated.VexJustificationComponentNotPresent,
> }
> ```

Oh, okay, thanks! I have updated the justification map and the status map.